From d8831dede08ca7f6aa44ca9a8bcb96819a9b63e6 Mon Sep 17 00:00:00 2001
From: Vasiliy Kovalev <kovalev@altlinux.org>
Date: Sun, 21 Jul 2024 12:16:42 +0300
Subject: [PATCH] hwdb: fix MXC6655 accelerometer mount matrix for Aquarius Cmp
 NS483

Signed-off-by: Vasiliy Kovalev <kovalev@altlinux.org>
(cherry picked from commit 23f5f827562bc4a2a5c12ca906eeab2a7d497d3c)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index f7892680bc..21f4380dd0 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -117,6 +117,9 @@ sensor:modalias:acpi:BOSC0200*:dmi:*svnAcer*:*pnSpinSP111-34:*
 #########################################
 
 # Aquarius Cmp NS483
+sensor:modalias:acpi:MXC6655*:dmi:*:svnAquarius*:pnCmp*NS483:*
+ ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, -1, 0; 0, 0, 1
+
 sensor:modalias:acpi:MXC4005*:dmi:*:svnAquarius*:pnCmp*NS483:*
 # Aquarius NS483
 sensor:modalias:acpi:MXC6655*:dmi:*:svnAquarius*:pnNS483:*
